### ISSUE TITLE
[BUG] Can't setup the isolationLevel to Serializable

### ISSUE THREAD

**Issue by Kiran-G1** [delta-io/delta#1265]
Hello,

I'm trying to setup the isolation level to Serializable on open source delta using azure synapse notebook.

Command :
`ALTER TABLE  publish.testOptimization012 SET TBLPROPERTIES ('delta.isolationLevel' = 'Serializable')`

It seems like delta is not able to identify the configuration

`Error: Unknown configuration was specified: delta.isolationLevel
org.apache.spark.sql.delta.DeltaErrors$.unknownConfigurationKeyException(DeltaErrors.scala:398)
org.apache.spark.sql.delta.DeltaConfigsBase.$anonfun$validateConfigurations$3(DeltaConfig.scala:147)
scala.Option.getOrElse(Option.scala:189)
org.apache.spark.sql.delta.DeltaConfigsBase.$anonfun$validateConfigurations$1(DeltaConfig.scala:147)
scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:238)`

The same command works in databricks delta. Is this expected ? Why is this inconsistency between databricks delta and open delta ?

**Comment by dhia-gharsallaoui** [comment]
Hello @Kiran-G1 ,
I don't think it's a bug. It's not implemented in the open source version. You can check this issue which describes well `Expose Isolation Level` #1261 .

**Comment by allisonport-db** [comment]
Thanks for reporting this, you're correct `'delta.isolationLevel' = 'Serializable'` should be valid.

As a workaround for now, in Delta 2.0 (coming soon) you will be able to disable the table property check with `spark.databricks.delta.allowArbitraryProperties.enabled=true` in the Spark session configuration.

**Comment by koertkuipers** [comment]
@allisonport-db i dont think that disabling that check will solve the issue? the codepath to set isolation level is simply not present in oss delta. see https://github.com/delta-io/delta/issues/1261

**Comment by koertkuipers** [comment]
@allisonport-db oh i got it know... you are saying let it be set despite fact that it doesn't do anything.
